storage-framework/src/github-adapter/GithubFileEntry.ts

```diff
@@ -201,8 +200,8 @@ export class GithubFileEntry implements StorageFrameworkFileEntry {
     return new Result(async (resolve, reject) => {
```

Comment:
resolve & reject must always be called even if their types are void (=> then resolve()). 
Reason:
Without resolve, await does not work e.g.:
```js
await removeGithubFile(removeFileFullPath, sha)
// this line will be never called
console.log("after await")
```